app/components/press-release/form.js

```diff
@@ -26,4 +41,19 @@ export default class PressReleaseFormComponent extends Component {
     keyWordArray.splice(index, 1);
     this.args.pressRelease.keywords = [...keyWordArray];
   }
+
+  @action
+  setGovernmentFields(selectedOptions) {
+    this.args.pressRelease.governmentFields = selectedOptions;
+  }
+
+  @action
+  setPublicationChannels(channel) {
+    const index = this.selectedPublicationChannels.indexOf(channel);
+    if (index > -1) {
+      this.selectedPublicationChannels.removeObject(channel);
+    } else {
+      this.selectedPublicationChannels.addObject(channel);
+    }
+  }
```

Comment:
Kan je hier de interface van `@onChange` van `InputField::PressReleasePublicationChannels` nog aanpassen zodanig dat je als argument een lijst van de geselecteerde publication channels krijgt ipv enkel het publication channel waarvan de checkbox gewijzigd is. Dan is de interface volledig hetzelfde als bij `<InputField::GovernmentFieldSelect>` en moet je hier enkel uitvoeren:
```
this.args.pressRelease.publicationChannels = selectedChannels;
```